css/properties/font-style.json

```diff
@@ -0,0 +1,59 @@
+{
+  "css": {
+    "properties": {
+      "font-style": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/font-style",
+          "support": {
+            "webview_android": {
+              "version_added": "1"
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": null
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1",
+              "notes": "Before Firefox 44, <code>oblique</code> was not distinguished from <code>italic</code>."
+            },
+            "firefox_android": {
+              "version_added": "4",
+              "notes": "Before Firefox 44, <code>oblique</code> was not distinguished from <code>italic</code>."
+            },
+            "ie": {
+              "version_added": "4"
+            },
+            "ie_mobile": {
+              "version_added": "6"
+            },
+            "opera": {
+              "version_added": "7"
+            },
+            "opera_android": {
+              "version_added": "6"
```

Comment:
This is another mobile-before-desktop case that I was suspicious of. Is it documented anywhere when the various mobile/desktop browsers get their version numbers synced up? I think Firefox, Chrome, and Opera do this at different versions, but I don't know when.

Reply:
This discrepency is correct: see http://www.opera.com/docs/history/presto/ and Opera Mobile history on wikipedia.

These are the first releases supporting the Presto engine.